eval/src/vespa/eval/eval/tensor_instructions.h

```diff
@@ -0,0 +1,24 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include "interpreted_function.h"
+
+namespace vespalib {
+
+class Stash;
+
+namespace eval {
+
+struct JoinPlan;
```

Comment:
There is no "JoinPlan".

Reply:
it got renamed to JoinParam and made local, I will clean it up